Sources/MapboxMaps/Foundation/Camera/CameraAnimator.swift

```diff
@@ -1,20 +1,40 @@
 import UIKit
+import CoreLocation
 
-// MARK: CameraAnimator Class
-public class CameraAnimator: NSObject {
+@objc public protocol CameraAnimatorProtocol: AnyObject {
```

Comment:
same transformation here with moving `public` to members

Reply:
Can't do that for @objc protocols unfortunately. I think we should remove this as part of the public interface.. Just not sure if we should do it as part of this PR.